lib/src/test/java/com/tbruyelle/rxpermissions3/RxPermissionsTest.java

```diff
@@ -513,7 +488,6 @@ public void eachSubscriptionCombined_trigger_granted() {
         mRxPermissions.onRequestPermissionsResult(new String[]{permission}, result);
 
         sub.assertNoErrors();
-        sub.assertNotTerminated();
```

Comment:
why these deletions?

Reply:
Hi @epool ,

Those methods were removed in RxJava 3 from TestObservable. See here:
https://github.com/ReactiveX/RxJava/wiki/What's-different-in-3.0#test-support-methods

Reply:
got it, thanks!